instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/example/.env

```diff
@@ -0,0 +1,10 @@
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318
+OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+OTEL_SERVICE_NAME=opentelemetry-python-openai
+
+# Change to 'false' to disable logging
+OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED=true
+# Change to 'console' if your OTLP endpoint doesn't support logs
+OTEL_LOGS_EXPORTER=otlp_proto_http
```

Comment:
nit:

any reason to not use defaults?
openai example does this now

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/5dfe192f9af588d33b67053d9a5d85f2f175f5ae/instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/manual/.env#L9-L12

Reply:
Incorporated into https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3123